common/browser_context_options.go

```diff
@@ -232,3 +232,29 @@ func (w *WaitForEventOptions) Parse(ctx context.Context, optsOrPredicate goja.Va
 
 	return nil
 }
+
+// GrantPermissionsOptions are the options used by the browserContext.grantPermissions
+// API.
```

Comment:
Since this comment appears on pkg.go.dev. Is it better to refer to Go-specific types instead of the JavaScript API? Maybe like: `// GrantPermissionsOptions are used by BrowserContext.GrantPermissions`? Also, there's work going on on the Go side to make these references clickable on the documentation (soon).

Reply:
Yeah, good point, i'll change that.

Resolved in cf5a95cf9ac1f4fd02374ae29b86b4fa8357fc2b